### DISCUSSION TITLE
How create or edit a layout?

### DISCUSSION THREAD

**Discussion by youngkaneda** [herbstluftwm/herbstluftwm#1494] [Q&A] [answered]
I wanna make my own layout, making the frames spawn according to my own strategy, but I couldn't find a way or where to do it.

**Comment by mgutz** [comment, marked as answer]
Not sure if this is the answer to your question. `herbstclient dump` will dump the current layout to the terminal. That will print an s-expr of the layout. You can safely remove client IDs from the output and restore with `herbstclient load s-expr`. Shouldn't be too hard to create your own s-expr for your strategies once you dumped a few layouts to see the format.

**Reply by youngkaneda** [reply]
Hey bro, thx, didn't thought about that, I was going with a keybind to call a function that create all the frames, set the attributes that I need to reproduce the layout, but your solution is way easier, thx a lot!!
